theLudovyc/TheStorage.gd

```diff
@@ -50,18 +50,8 @@ func try_to_sell_resource(resource_type: Resources.Types, amount: int) -> bool:
 	return false
 
 
-func update_global_production_rate(resource_type: Resources.Types):
```

Comment:
If you remove it, how do you send the resource update ? 🤔

Reply:
I mean the resource production rate per tick, with event_bus.resource_prodution_rate_updated